sdk/state/close.go

```diff
@@ -98,8 +108,7 @@ func (c *Channel) ConfirmCoordinatedClose(cc CoordinatedClose) (CoordinatedClose
 		cc.closeSignatures = append(cc.closeSignatures, txCoordinatedClose.Signatures()...)
 	}
 
-	// TODO - merge instead of overwrite, similar to ConfirmProposal
-	c.coordinatedClose = cc
+	c.coordinatedClose = mergeCoordinatedCloseData(c.coordinatedClose, cc)
```

Comment:
We need this same behavior on all the other confirm functions too. Not sure if you're planning that for a separate PR, but I think you could do it here too.

Reply:
I added for the `Update` and `Close` confirm methods. I believe those are the only confirm methods it's needed atm

For the `Open` steps we're actually not storing signatures atm. I added this ticket: https://github.com/stellar/experimental-payment-channels/issues/92 for changing that to match the `Update` and `Close` steps. For that ticket I can add this `appendNewSignatures` functionality (assuming you agree the `Open` should start storing sigs, if you could confirm in that ticket that would be great 😄 )